traits/ctraits.c

```diff
@@ -100,21 +77,6 @@ do { \
     ((((tnotifiers) != NULL) && (PyList_GET_SIZE((tnotifiers))>0)) || \
      (((onotifiers) != NULL) && (PyList_GET_SIZE((onotifiers))>0)))
 
-/* Python version dependent macros: */
-#if ( (PY_MAJOR_VERSION == 2) && (PY_MINOR_VERSION < 3) )
-#define PyMODINIT_FUNC void
-#define PyDoc_VAR(name) static char name[]
-#define PyDoc_STRVAR(name,str) PyDoc_VAR(name) = PyDoc_STR(str)
-#ifdef WITH_DOC_STRINGS
-#define PyDoc_STR(str) str
-#else
-#define PyDoc_STR(str) ""
-#endif
-#endif
-#if (PY_VERSION_HEX < 0x02050000)
-typedef int Py_ssize_t;
```

Comment:
i'm not a sure where these macros are defined in the Python source header files. Still looking for them.

Reply:
`PyMODINIT_FUNC`: https://github.com/python/cpython/blob/8d62df60d8733d0fa9aee14ef746d0009a7a9726/Include/pyport.h#L670

`Py_ssize_t`: https://github.com/python/cpython/blob/8d62df60d8733d0fa9aee14ef746d0009a7a9726/Include/pyport.h#L104-L110

The `PyDoc_*` stuff is in `pymacro.h`.

Reply:
should I explicitly use `#include "pyport.h"` and `#include "pymacro.h"` with comments about what we're including them for or should I just use the implicit imports from `# include "python.h"`?

I don't know if there's a cpython-recommended way of doing this and i didn't find anything relevant in PEP7. (The former sounds better to me because it's explicit instead of implicit).

Reply:
No, you shouldn't need any Python-related includes beyond #include "Python.h". That include file includes everything else it needs.

Reply:
Thanks. Merging.